packages/astro/src/runtime.ts

```diff
@@ -250,6 +255,18 @@ async function load(config: RuntimeConfig, rawPathname: string | undefined): Pro
       };
     }
 
+    if (err instanceof ReferenceError && err.toString().includes('window is not defined')) {
+      return {
+        statusCode: 500,
+        type: 'ssr',
+        error: new Error(
+          `[${reqPath}]
+    The window object is not available during server-side rendering.
+    https://github.com/snowpackjs/astro/blob/main/docs/reference/api-reference.md#importmeta`
```

Comment:
WDYT about this error message? Should it say more?

Reply:
I like it overall. Maybe an additional sentence like: `import.meta.env.SSR can be used to to prevent running browser code.` (write a better sentence than that though 😀)